kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/control/paintpicker/colorpicker/ColorPicker.java

```diff
@@ -252,6 +256,11 @@ private void initialize() {
         picker_region.pressedProperty().addListener(liveUpdateListener);
         hue_slider.pressedProperty().addListener(liveUpdateListener);
         alpha_slider.pressedProperty().addListener(liveUpdateListener);
+
+        // Fix ColorPicker movement on focus. Now blocks scroll and key events (Home, End, PgUp, PgDn, arrows) to keep it static. For issue #654.
```

Comment:
Looks good, just a small rewording:

```suggestion
        // Disable ColorPicker movement on focus, by blocking scroll and key (Home, End, PgUp, PgDn, arrows) events
```